heareval/predictions/task_predictions.py

```diff
@@ -766,13 +792,14 @@ def __init__(
         self.trainer = trainer
         self.validation_score = validation_score
         self.score_mode = score_mode
+        self.conf = conf
 
 
 def task_predictions_train(
     embedding_path: Path,
     embedding_size: int,
-    grid_points: int,
     metadata: Dict[str, Any],
+    data_splits: Dict[str, List[str]],
```

Comment:
What is data_splits? Maybe it's worth having a doc-string here? Or is this called by another function with the same set of parameters?

I am wondering if it even makes sense that we have a class for such a long list of things? Or most of them if it makes sense to divide it in a sensible way?

Reply:
#319